src/CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageQueueTransport.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+using CoreWCF.Configuration;
+using CoreWCF.Queue;
+using CoreWCF.Queue.Common;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace CoreWCF.Channels
+{
+    public class AzureQueueStorageQueueTransport : IQueueTransport
+    {
+        private readonly string _connectionString;
+        private readonly string _queueName;
+        private readonly string _deadLetterQueueConnectionString;
+        private readonly string _deadLetterQueueName;
+        private readonly QueueClient _queueClient;
+        private readonly QueueClient _deadLetterQueueClient;
+        private readonly TimeSpan _queueReceiveTimeOut;
+        private readonly TimeSpan _receiveMessagevisibilityTimeout;
+        private readonly ILogger<AzureQueueStorageQueueTransport> _logger;
+
+        public AzureQueueStorageQueueTransport(IServiceDispatcher serviceDispatcher, IServiceProvider serviceProvider)
+        {
+            _queueClient = new QueueClient(_connectionString, _queueName);
+            _deadLetterQueueClient = new QueueClient(_deadLetterQueueConnectionString, _deadLetterQueueName);
```

Comment:
Where are _connectionString and  _deafLetterQueueConnectionString getting set? Also are we going to require a dead letter queue?

Reply:
Sorry i thought i updated this code but may be i somehow got removed, I no longer require the connection string as the object will be got using DI. regarding deadletter queue, i thought we wanted one to store the messages which can't be processed,

Reply:
Pulling it from DI means you are potentially using the same DeadLetterQueue instance for multiple services and/or endpoint which might not be appropriate. While a dead letter queue is useful, it might not always be wanted. It's extra administrative overhead if you have 100% control of the client and the server with really low risk of bad messages. For others who have the client. It should be opt-outable and configurable on a per-binding instance.